src/EnergyPlus/ChillerAbsorption.hh

```diff
@@ -54,51 +54,54 @@
 // EnergyPlus Headers
 #include <EnergyPlus/DataGlobals.hh>
 #include <EnergyPlus/EnergyPlus.hh>
+#include <EnergyPlus/PlantComponent.hh>
 
 namespace EnergyPlus {
 
 namespace ChillerAbsorption {
 
-    // Using/Aliasing
-
-    // Data
-    // MODULE PARAMETER DEFINITIONS:
-    // chiller flow modes
     extern int const FlowModeNotSet;
     extern int const ConstantFlow;
     extern int const NotModulated;
     extern int const LeavingSetPointModulated;
 
-    // DERIVED TYPE DEFINITIONS:
-
-    // MODULE VARIABLE DECLARATIONS:
-    extern int NumBLASTAbsorbers; // number of Absorption Chillers specified in input
-
-    extern Real64 CondMassFlowRate;    // Kg/s - condenser mass flow rate, water side
-    extern Real64 EvapMassFlowRate;    // Kg/s - evaporator mass flow rate, water side
-    extern Real64 SteamMassFlowRate;   // Kg/s - steam mass flow rate, water side
-    extern Real64 CondOutletTemp;      // C - condenser outlet temperature, water side
-    extern Real64 EvapOutletTemp;      // C - evaporator outlet temperature, water side
-    extern Real64 GenOutletTemp;       // C - generator fluid outlet temperature
-    extern Real64 SteamOutletEnthalpy; // J/kg - generator fluid outlet enthalpy
-    extern Real64 PumpingPower;        // W - rate of Absorber energy use
-    extern Real64 PumpingEnergy;       // J - Absorber energy use
-    extern Real64 QGenerator;          // W - rate of Absorber steam use
-    extern Real64 GeneratorEnergy;     // J - Absorber steam use
-    extern Real64 QEvaporator;         // W - rate of heat transfer to the evaporator coil
-    extern Real64 EvaporatorEnergy;    // J - heat transfer to the evaporator coil
-    extern Real64 QCondenser;          // W - rate of heat transfer to the condenser coil
-    extern Real64 CondenserEnergy;     // J - heat transfer to the condenser coil
+    extern int numBlastAbsorbers; // number of Absorption Chillers specified in input
 
     extern Array1D_bool CheckEquipName;
 
-    extern bool GetInput; // When TRUE, calls subroutine to read input file
+    extern bool getInput; // When TRUE, calls subroutine to read input file
```

Comment:
Note for later, a lot of times these getInput flags aren't actually needed to be defined and exposed via the header file, it can be defined at the top of the cc file only, so just watch out for that on later refactors.  Note there are occasions where this isn't actually true, so I'm just throwing this out there.

Reply:
@Myoldmopar thanks. This is externed because FaultsManager uses it.